tests/unit/test_clidocs.py

```diff
@@ -35,12 +42,6 @@ def setUp(self):
         self.operation_handler = OperationDocumentEventHandler(
             self.help_command)
 
-    def assert_rendered_docs_contain(self, expected):
```

Comment:
What about just extracting a base test class with this assertion method so the two test classes can share?

assertIn() will show the expected and the contents in the error message on test failure.  With `assertTrue(_rendered_docs_contains)` we'll just get "Assertion error: False is not True".